src/pool/srv_pool.c

```diff
@@ -1308,6 +1308,12 @@ read_db_for_stepping_up(struct pool_svc *svc, struct pool_buf **map_buf,
 	return rc;
 }
 
+int
+ds_pool_svc_rf_to_nreplicas(int svc_rf)
```

Comment:
callers should be careful to not pass svc_rf < 0 here

or, could this function recognize that case and calculate the objective according to a default (2) that would be defined either local to this function, or made more broadly available to the pool code?